example-code/app/package.json

```diff
@@ -28,7 +28,7 @@
   "devDependencies": {
     "@itwin/build-tools": "workspace:*",
     "@itwin/eslint-plugin": "5.2.1",
-    "@itwin/oidc-signin-tool": "^4.4.1",
+    "@itwin/oidc-signin-tool": "^5.0.2",
```

Comment:
idk if we should be pulling in 5.x oidc-signin-tool into the repo here if it pulls in 5.x of core

Reply:
are we using `^` or `~` with playwright in oidc-signin-tool?